extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/runtime/RedisClientUtil.java

```diff
@@ -0,0 +1,63 @@
+package io.quarkus.redis.client.runtime;
+
+import java.net.URI;
+import java.util.Set;
+
+import io.quarkus.redis.client.runtime.RedisConfig.RedisConfiguration;
+import io.quarkus.runtime.configuration.ConfigurationException;
+import io.vertx.redis.client.RedisClientType;
+import io.vertx.redis.client.RedisOptions;
+
+public class RedisClientUtil {
+    public static final String DEFAULT_CLIENT = "__redis_default_client__";
```

Comment:
In the other extensions, we call it `<default>`.

Reply:
Sure, let's be consistent here too. I'll change it.